zstacklib/zstacklib/utils/ovn.py

```
@@ -5,6 +5,9 @@
 import os
 import shutil
 import time
+from os.path import split
```

Ruff (0.11.9)

`os.path.split` imported but unused

Remove unused import: `os.path.split`

(F401)